package.json

```diff
@@ -125,9 +124,12 @@
         "webpack": "^4.43.0"
     },
     "dependencies": {
+        "@storybook/addon-controls": "^6.2.8",
+        "@storybook/addon-postcss": "^2.0.0",
         "dayjs": "^1.8.10",
         "patch-package": "^6.4.6",
         "react-keyed-flatten-children": "^1.3.0",
+        "react-markdown": "^5.0.3",
```

Comment:
I think these are all used in stories only, while it might trigger an eslint warning if we use them in a non-test module with these being `devDependencies` I think we should still move them over there so they don't get pulled down when installing reactist.